draft-shafranovich-rfc4180-bis.md

```diff
@@ -136,6 +136,16 @@ another double quote. For example:
 
    "aaa","b""bb","ccc"CRLF
 
+8. A hash sign MAY be used to mark lines that are meant to be commented lines.
+A commented line can contain any whitespace or visible character until it is
```

Comment:
Why restrict only to visible characters and not allow Unicode?

Also, shouldn't "#" be listed above in section 6 as requiring double quotes?

Reply:
Also perhaps mention that not all implementations support comments?

Reply:
Unicode is not listed because #13 is still a thing. You're right about 6.

Reply:
Ok so we will fix Unicode here, once #13 is resolved

Reply:
Done with section 6.